posts/app.py

```diff
@@ -19,11 +19,16 @@
 # limitations under the License.
 
 import os
+import uuid
 import pymongo
 from flask import Flask, jsonify, request
+from werkzeug.exceptions import BadRequestKeyError
+from google.cloud import storage
 
 app = Flask(__name__)  # pylint: disable=invalid-name
 
+GCLOUD_STORAGE_BUCKET_NAME = "large_events_posts_service_bucket"
+
```

Comment:
Remove extra line

Reply:
[PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines) says "Surround top-level function and class definitions with two blank lines."